source/utils/date.cpp

```diff
@@ -120,7 +105,7 @@ int date::to_number(calendar base_date) const
 
 date date::today()
 {
-    std::tm now = safe_localtime(std::time(nullptr));
+    std::tm now = detail::localtime_safe(std::time(nullptr));
```

Comment:
Some particular reason why you switched `safe_localtime` to `localtime_safe`?

Reply:
Personally, I prefer seeing as soon as possible which function I'm actually calling. In my own projects, I prefer using a suffix like `_safe` instead of a prefix, because this way I can see more easily the difference between e.g. `localtime_safe` and `gmtime_safe`. But this is just my preference :smile: We can do it differently - I don't mind, as long as it's easily understandable by everyone and makes sense!

Reply:
Ok, fine for me.